boards/arm/frdm_kw41z/Kconfig.defconfig

```diff
@@ -85,4 +85,18 @@ config FXOS8700_DRDY_INT1
 
 endif # FXOS8700
 
+if NETWORKING
+
+config NET_L2_IEEE802154
+	def_bool y
+
+config IEEE802154_KW41Z
+	def_bool y
+
+# Should match IEEE802154_KW41Z_DRV_NAME
+config NET_APP_IEEE802154_DEV_NAME
+	default "KW41Z"
```

Comment:
instead of the comment, why not just do
```default IEEE802154_KW41Z_DRV_NAME```

Reply:
I'm afraid to push Kconfig too far and then need to debug that ;-). That worked though.